build-support/bin/cherry_pick.sh

```diff
@@ -51,6 +51,7 @@ COMMIT=$(gh pr view "$PR_NUM" --json mergeCommit --jq '.mergeCommit.oid')
 if [[ -z $COMMIT ]]; then
   fail "Wasn't able to retrieve merge commit for $PR_NUM."
 fi
+git fetch https://github.com/pantsbuild/pants "$COMMIT"
```

Comment:
This means the person running the script doesn't need an up-to-date `main`, we'll fetch the ref.